File: src/main/scala/com/github/mauricio/postgresql/messages/PreparedStatementExecuteMessage.scala
package com.github.mauricio.postgresql.messages

import com.github.mauricio.postgresql.Message

/**
 * User: Maurício Linhares
 * Date: 3/12/12
 * Time: 6:50 PM
 */

class PreparedStatementExecuteMessage( query : String, values : Seq[Any] )
  extends PreparedStatementMessage(Message.Execute, query, values)

File: postgresql-async/src/main/java/com/github/jasync/sql/db/postgresql/messages/frontend/PreparedStatementExecuteMessage.kt
package com.github.jasync.sql.db.postgresql.messages.frontend

import com.github.jasync.sql.db.column.ColumnEncoderRegistry
import com.github.jasync.sql.db.postgresql.messages.backend.ServerMessage
import java.util.UUID

class PreparedStatementExecuteMessage(
        statementId: UUID,
        query: String,
        values: List<Any?>,
        encoderRegistry: ColumnEncoderRegistry
) : PreparedStatementMessage(statementId, ServerMessage.Execute, query, values, encoderRegistry)
